core/input/input_map.h

```diff
@@ -58,7 +58,7 @@ class InputMap : public Object {
 	OrderedHashMap<String, List<Ref<InputEvent>>> default_builtin_cache;
 	OrderedHashMap<String, List<Ref<InputEvent>>> default_builtin_with_overrides_cache;
 
-	List<Ref<InputEvent>>::Element *_find_event(Action &p_action, const Ref<InputEvent> &p_event, bool p_exact_match = false, bool *p_pressed = nullptr, float *p_strength = nullptr, float *p_raw_strength = nullptr) const;
+	List<Ref<InputEvent>>::Element *_find_event(Action &p_action, const Ref<InputEvent> &p_event, const bool p_exact_match = false, bool *const p_pressed = nullptr, float *const p_strength = nullptr, float *const p_raw_strength = nullptr) const;
```

Comment:
Primitive types (`int`, `bool`, `float`, etc.) shouldn't be made `const` in the prototypes.